projects/demo/src/modules/experimental/card/examples/7/index.less

```diff
@@ -0,0 +1,36 @@
+@import 'taiga-ui-local';
+
+:host {
+    display: flex;
+    gap: 1rem;
+}
+
+.surface[data-surface],
+.surface[data-surface] {
```

Comment:
Unnecessary duplication. Do we really need that specificity bump, btw? I suppose it's because size is under this `&[data-size='m']`, but we can do it differently, define `width: var(--t-size)` on host and override the variable under `data-size`. This way we can use just `.surface` here.

Reply:
fixed